clients/admin-ui/cypress/e2e/datasets-classify.cy.ts

```diff
@@ -36,19 +36,6 @@ describe("Datasets with Fides Classify", () => {
       cy.getByTestId("input-classify").find("input").should("not.be.checked");
     });
 
-    it("Can render the 'Status' column and classification status badges in the dataset table when plus features are enabled", () => {
```

Comment:
Maybe it was something in the placement of the badge? I was thinking it might have been because the status being tested was unknown (since the test wasn't inclusive of `fidesplus`) but will dive a bit deeper to try to keep the test

Reply:
Put back! Mostly :) That test was looking for the generated state to be Unknown, which we removed. I flipped the test to validate the badge does not exist at that point and then added an existence check further down. Hoping that works!